### DISCUSSION TITLE
Compressible flow simulation on a payload

### DISCUSSION THREAD

**Discussion by Ajaykrishna0703** [byuflowlab/FLOWUnsteady#212] [Q&A]
Dear Community members
I have a payload that needs to be simulated in FlowUnsteady for quick results. I plan to generate the payload geometry in OpenVSP and import the DegenGeom into FlowUnsteady. I have a few questions:

1.The payload geometry is fairly complex and resembles a bluff body. Can FlowUnsteady handle such geometries effectively?
2.The freestream Mach number is 0.34, placing it in the compressible flow regime. Can FlowUnsteady manage this flow speed?"

**Comment by Yikun00** [comment]
Hello, I have encountered a similar compressible simulation requirement. Can FLOWUnsteay implement this requirement?
